include/mbedtls/ctr_drbg.h

```diff
@@ -141,16 +141,16 @@ mbedtls_ctr_drbg_context;
 void mbedtls_ctr_drbg_init( mbedtls_ctr_drbg_context *ctx );
 
 /**
- * \brief               This function seeds and sets up the CTR_DRBG 
+ * \brief               This function seeds and sets up the CTR_DRBG	
```

Comment:
Blocker: Please remove the trailing whitespace.

Reply:
done